src/cryptonote_basic/hardfork.cpp

```diff
@@ -90,6 +91,7 @@ static constexpr HardFork::Params testnet_hard_forks[] =
   { network_version_15_lns,                 244777, 0, 1583940000 }, // 2020-03-11 15:20UTC
   { network_version_16_pulse,               382222, 0, 1600468200 }, // 2020-09-18 22:30UTC
   { network_version_17,                     447275, 0, 1608276840 }, // 2020-12-18 05:34UTC 
+  { network_version_18,                     447275, 0, 1608276840 }, // 2020-12-18 05:34UTC 
```

Comment:
The current testnet syncs up to 501743, so let's set this to 501750.  Timestamp can be the current time to be lazy: `1616631051`